internal/controller/eventbus/controller.go

```diff
@@ -58,6 +58,8 @@ var (
 
 const (
 	maximumEventlogNum = 64
+	mappingKey         = "__%s_%s__" // __{namespace}_{eventbus}__
```

Comment:
namespace or eventbus name can use _ 
so it needs to use other character, such as %